### ISSUE TITLE
Buckets without subfolder

### ISSUE THREAD

**Issue by felixbecker** [moriyoshi/s3-sftp-proxy#5]
Hi, 

how do I have to specify my bucket settings in the `s3-sftp-proxy.toml` when I just want to use the whole bucket as sftp backend without prefixes?

I tried so far:

```
bucket = "my-bucket-name"
key_prefix = "/" 
```

```
bucket = "my-bucket-name"
key_prefix = "" 
```

```
bucket_url = "s3://my-bucket-name/" 
```

```
bucket_url = "s3://my-bucket-name" 
```

when I enable debug I get also an error message:

```
INFO[0000] Listen on :10022
INFO[0028] connected from client 127.0.0.1:64876
INFO[0031] user user01 logged in
INFO[0031] channel: session
DEBU[0034] S3ObjectStat.ListAt: len(result)=1 offset=0
DEBU[0034] GetObjectAclWithContext(Bucket=my-bucket-name, Key=)
DEBU[0034] {GetObjectAclWithContext: bucket: my-bucket-name' key: '' InvalidParameter: 1 validation error(s) found.
- minimum field size of 1, GetObjectAclInput.Key.

DEBU[0034] => InvalidParameter: 1 validation error(s) found.
- minimum field size of 1, GetObjectAclInput.Key.
```

running the server locally on my macbook gives me also 
the following error:

```
$ sftp -P 10022 user01@localhost
user01@localhost's password: 
Connected to localhost.
sftp> ls 
remote readdir("/"): No such file or directory
sftp> 

```

What can I do to get this running? 

When I create a folder on s3 and prefixes it at least kind of works but shows the folder as file in my sftp view.
And btw: the issue: https://github.com/moriyoshi/s3-sftp-proxy/issues/3 is still present running the server on macos.

**Comment by moriyoshi** [comment]
That seems to be a bug.  I'll take a look.

**Comment by moriyoshi** [comment]
This should be fixed in dc76247.  Check it out.

**Comment by felixbecker** [comment]
Hi, thanks for fixing! It works like a charm! Maybe could also take a look at the PR. This is just a small change which should not affect running code and would help me in my deployment scenario. 
Thanks again!
